server/preprocessing/conf/config.ini

```diff
@@ -3,15 +3,18 @@
 
 [general]
 # Full path to the preprocessing directory
-preprocessing_dir = "/home/rbachleitner/workspace/ausleihhistory/kc_frontend/vis/js/headstart/server/preprocessing/"
+preprocessing_dir = "/home/rbachleitner/workspace/Headstart_new/server/preprocessing/"
 # Full path to the images directory for the client. Needs to be in the public_html/www directory. Make sure that your webserver has write access to this directory.
-images_path = "/"
+images_path = "/home/rbachleitner/workspace/Headstart_new/server/paper_preview/"
 # Host of the client visualization
-host = "localhost/"
+host = "http://localhost/"
 # Relative path to the client visualization. Needs to be in the public_html/www directory.
-vis_path = "headstart3/"
+vis_path = "examples/search_repos/"
 # Relative path to the client REST services. Needs to be in the public_html/www directory.
-services_path = "headstart3/server/services/"
+services_path = "server/services/"
+phantomjs_path = "/home/rbachleitner/Downloads/phantomjs-2.1.1-linux-x86_64/bin/phantomjs"
```

Comment:
Please create an own section "snapshot" for these settings and provide a description for them.